### PR TITLE
fix(installation: installer_destopselection): case for other desktop

### PR DESCRIPTION
**Pull request by actionless**

### CODE REVIEW

File: tests/installation/installer_desktopselection.pm [hunk 1]
```diff
@@ -24,8 +25,20 @@ sub run() {
...
+        }
+        elsif ( $d eq "lxde" ) {
+            assert_screen "lxde-selected", 3;
+        }
+        elsif ( $d eq "minimalx" ) {
+            assert_screen "minimalx-selected", 3;
+        }
+        elsif ( $d eq "textmode" ) {
+            assert_screen "textmode-selected", 3;
+        }
```
Comment (actionless) [inline tests/installation/installer_desktopselection.pm]: or better `assert_screen "$d-selected"`?

**Comment by nilxam** [comment]
hi, I can't get the exact bug what be fixed here, please check https://openqa.opensuse.org/tests/overview?distri=opensuse&version=Tumbleweed&build=20150524&groupid=1 , we did xfce,lxde, minimalx and textmode test and they're green on openqa.opensuse.org. Maybe you've wrong DESKTOP var sets on your template?

**Comment by actionless** [comment]
@nilxam Yup, it's for a case when desktop is none of those, like `icewm`, `openbox` or whatever

Also later i found the same problem in `/console/consoletest_finish.pm` (no fallback value too, what makes to fail all the later tests). Should i add fix for it to this PR or such changes are not needed at all?

**Comment by nilxam** [comment]
hmm...openSUSE installer didn't provide the desktop option like icewm or openbox currently, and we already handled all desktop options what we knew ie. (kde => "k", gnome => "g", xfce => "x", lxde => "l", minimalx => "m", textmode => "i"), thus I can't see the use case for such changes...

**Comment by aaannz** [comment]
Yeah, but I guess if you want to test other WMs, you need to select minimal x during installation and later install your WM packages. So I see the use case here. And openbox, awesome, etc. are in our repos.

**Comment by nilxam** [comment]
yes, in case test other WMs, but suppose it has a "openbox" test, IMO it should sets DESKTOP=minimalx and WM=openbox in its testsuite, and use WM var in main.pm or in whatever tests if needed :)

**Comment by actionless** [comment]
@nilxam thanks for suggestion, i will try to do it like that

**Pull request closed without merging**